src/components/common/icon-button/icon-button.styles.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components";
+
+interface IconButtonStyleProps {
+    $iconSize?: string;
+    $iconColor?: (props: any) => string;
+}
+
+export const IconButton = styled.button<IconButtonStyleProps>`
+    padding: 0;
+    border: none;
+    display: grid;
```

Comment:
this style and the one below does nothing since there is no height or width specified. In other words, the button is taking the size of the svg. There is no centering required